setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+with open("requirements.txt") as requirements:
+    requirements = requirements.readlines()
+
+setup(
+    name='versations',
+    version='0.1.0',
+    py_modules=['main.py'],
```

Comment:
This results in a project that only contains `main.py`, which I do not think is what you want.  The code should instead be placed under a package directory (say, `versations/`) which is then specified to `setup()` via `packages=["versations"]`.

Reply:
Ah ok, that explains some weird behavior. (It seemed to require -e, but might have just needed to be run from the source directory)